src/providers/generic/zip/AdmZipProvider.ts

```diff
@@ -27,16 +23,11 @@ export default class AdmZipProvider extends ZipProvider {
 
     async extractEntryTo(zip: string | Buffer, target: string, outputPath: string): Promise<void> {
         const adm = new AdmZip(zip);
-        return this.sanitizedExtraction(adm.getEntry(target)!, outputPath);
-    }
-
-    private async sanitizedExtraction(entry: IZipEntry, outputPath: string): Promise<void> {
-        const sanitizedTargetName = entry.entryName.split('\\').join('/');
-        await FileUtils.ensureDirectory(path.dirname(path.join(outputPath, sanitizedTargetName)));
-        if (entry.isDirectory)
-            await FileUtils.ensureDirectory(path.join(outputPath, sanitizedTargetName));
-        else
-            await FsProvider.instance.writeFile(path.join(outputPath, sanitizedTargetName), entry.getData());
+        if(!path.posix.normalize(path.join(outputPath, target)).startsWith(outputPath))
```

Comment:
This supposedly doesn't normalize `\` characters. So zips created on Windows have the potential to be invalid on Linux if I'm not mistaken?

Reply:
I believe I tested it on Linux with a Windows-created zip, but here's a new commit to address edge cases just in case it does matter.